config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -113,9 +113,9 @@ periodics:
         - "./scripts/ci-e2e.sh"
       env:
         - name: KUBERNETES_VERSION_UPGRADE_FROM
-          value: "stable-1.24"
-        - name: KUBERNETES_VERSION_UPGRADE_TO
           value: "stable-1.25"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "stable-1.26"
         - name: ETCD_VERSION_UPGRADE_TO
           value: "3.5.3-0"
```

Comment:
etcd and coredns versions can be bumped too, see https://github.com/kubernetes/test-infra/commit/3107bae4e798ef5f3c949d7b4e2d3a6afc6b497e

Reply:
done